### DISCUSSION TITLE
How to include dotnet framework version as constraint

### DISCUSSION THREAD

**Discussion by omlnaut** [renovatebot/renovate#26627] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

We're currently on dotnet 7 and can't upgrade yet. Renovate gives us package updates that only work on dotnet 8.

Is there a way to configure (on a project level of globally) the framework version as constraint for updated package versions?

I already tried adding constraints.dotnet = 7.0.0 to the global renovate json, but that didn't have any effect.

Constraining the dotnet sdk version in the package rules to 7.0.0 does indeed prevent the SDK update, but other packages are still updated to versions that require dotnet 8.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by viceice** [comment]
not yet supported

https://docs.renovatebot.com/configuration-options/#constraintsfiltering

**Reply by espeneriksmoenloke** [reply]
Do you know if this is something that is supported now?
